ocpmodels/datasets/lmdb_dataset.py

```diff
@@ -144,6 +194,44 @@ def close_db(self):
         else:
             self.env.close()
 
+    def get_metadata(self, Nsamples=100):
+        # This will interogate the classic OCP LMDB format to determine
+        # which properties are present and attempt to guess their shapes
+        # and whether they are intensive or extensive.
+
+        # Grab an example data point
+        example_pyg_data = self.__getitem__(0)
+
+        # Check for all properties we've used for OCP datasets in the past
+        props = []
+        for potential_prop in [
+            "y",
+            "y_relaxed",
+            "stress",
+            "stresses",
+            "force",
+            "forces",
+        ]:
+            if hasattr(example_pyg_data, potential_prop):
+                props.append(potential_prop)
+
+        # Get a bunch of random data samples and the number of atoms
+        sample_pyg = [
+            self[i] for i in np.random.choice(self.__len__(), size=(Nsamples,))
```

Comment:
Probably want to sample without replacement here as well

Reply:
Resolved in [c890169](https://github.com/Open-Catalyst-Project/ocp/pull/513/commits/c8901697aa93606e002943c341311930e2bb26f0)